makefiles/default-channel.inc.mk

```diff
@@ -1,13 +1,17 @@
 # Set a custom channel if needed
-ifneq (,$(filter cc110x,$(USEMODULE)))          # radio is cc110x sub-GHz
-  DEFAULT_CHANNEL ?= 0
+ifneq (,$(DEFAULT_CHANNEL))
```

Comment:
This now changes the behavior for Sub-GHz radios, as it is always set to channel 26 (see `sys/include/net/ieee802154.h`). It was previously set to channel 5 if `at86rf212b` is included.

Reply:
[`IEEE802154_DEFAULT_SUBGHZ_CHANNEL`](https://github.com/RIOT-OS/RIOT/blob/master/sys/include/net/ieee802154.h#L127) still defaults to 5 if `DEFAULT_CHANNEL` is not set.

Reply:
Ah, sorry. Only on my first coffee for today ^^.